src/Common/test-runtime/LatestDependencies/project.json

```diff
@@ -1,74 +0,0 @@
-{
```

Comment:
Question only: did the automatic tool delete this.  Was it manually deleted?  Don't block on the question.